graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -547,6 +546,22 @@ class DgsSchemaProvider(
         }
     }
 
+    private fun findDirectives(applicationContext: ApplicationContext, runtimeWiringBuilder: RuntimeWiring.Builder) {
+        applicationContext.getBeansWithAnnotation(DgsDirective::class.java).forEach { (_, directiveComponent) ->
+            val annotation = directiveComponent::class.java.getAnnotation(DgsDirective::class.java)
+            when (directiveComponent) {
+                is SchemaDirectiveWiring ->
+                    if (StringUtils.hasText(annotation.name)) runtimeWiringBuilder.directive(
+                        annotation.name,
+                        directiveComponent
+                    )
+                    else runtimeWiringBuilder.directiveWiring(directiveComponent)
```

Comment:
```suggestion
                    if (StringUtils.hasText(annotation.name)) {
                        runtimeWiringBuilder.directive(
                            annotation.name,
                            directiveComponent
                        )
                    } else {
                        runtimeWiringBuilder.directiveWiring(directiveComponent)
                    }
```

Looking at the project, I think they preferr having if/else with curly braces {}. Personally I think its also better readable